subprojects/packagefiles/expat/meson.build

```diff
@@ -97,19 +96,15 @@ expat_static_args = []
 
 add_project_arguments('-DXML_ENABLE_VISIBILITY=1', language: 'c')
 add_project_arguments('-DHAVE_EXPAT_CONFIG_H', language: 'c')
-if cc.has_argument('-fno-strict-aliasing')
-  add_project_arguments('-fno-strict-aliasing', language: 'c')
-endif
+add_project_arguments(cc.get_supported_arguments('-fno-strict-aliasing'), language: 'c')
 if cc.get_argument_syntax() == 'msvc'
   add_project_arguments('-D_CRT_SECURE_NO_WARNINGS', '-wd4996', language: 'c')
 endif
 
 if host_machine.system() == 'windows'
   add_project_arguments('-DXMLCALL=WINAPIV', language: 'c')
-  if get_option('default_library') != 'static'
-    add_project_arguments('-DXMLIMPORT=__declspec(dllexport)', language: 'c')
-  else
-    add_project_arguments('-DXMLIMPORT=__declspec(dllimport)', language: 'c')
+  add_project_arguments('-DXMLIMPORT=__declspec(dllimport)', language: 'c')
```

Comment:
How is those macros supposed to work? https://github.com/libexpat/libexpat/blob/master/expat/lib/expat_external.h#L87. I see no dllexport anywhere.

Reply:
There's a def file.